src/Libraries/Nop.Services/Messages/MessageTokenProvider.cs

```diff
@@ -1304,6 +1304,7 @@ public virtual async Task AddCustomerTokensAsync(IList<Token> tokens, Customer c
         tokens.Add(new Token("Customer.FirstName", customer.FirstName));
         tokens.Add(new Token("Customer.LastName", customer.LastName));
         tokens.Add(new Token("Customer.VatNumber", customer.VatNumber));
+        tokens.Add(new Token("Customer.Company", customer.Company));

```

Comment:
Hi, @G3r4rd00! Unfortunately, these changes are unrelated to task #7598, and they already have a [PR](https://github.com/nopSolutions/nopCommerce/pull/756). Please roll them back so that I can merge the main code